tokio/src/runtime/task/join.rs

```diff
@@ -210,6 +210,16 @@ impl<T> JoinHandle<T> {
             }
         }
     }
+
+    /// Returns a new `AbortHandle` that can be used to remotely abort this task.
+    #[cfg(any(tokio_unstable, test))]
```

Comment:
Why is this being enabled w/ `test`? This is not a public API, so it seems ok, but I am not following the thread.

Also, in #4518, I added a `cfg_unstable!` macro.

Reply:
The reason it's enabled with `cfg(test)` is because I added `AbortHandle` to the `task_combinations.rs` tasks, where it's kind of tightly-coupled with the rest of the test. That _could_ be cfg flagged, but it would mean filling the test code with a bunch of `#[cfg(tokio_unstable)]` attributes --- unlike the `task.rs` tests, we can't just stick `#[cfg(tokio_unstable)]` on a couple of unit tests and be done.